code/game/objects/structures/mineral_doors.dm

```diff
@@ -15,6 +15,7 @@
 	var/state_open = FALSE
 	var/is_operating = FALSE
 	var/close_delay = -1 //-1 if does not auto close.
+	var/operatetime = 10 //This variable was made solely to adapt sleep(10) for the alien subtype of this kind of door in operate()
```

Comment:
```suggestion
	/// This variable was made solely to adapt sleep(10) for the alien subtype of this kind of door in operate()
	var/operatetime = 10 
```
also this would probably be better as a define

Reply:
Removed that bit since I ended up making the resin doors be their own things instead of a subtype